cmd/policy/policy_test.go

```diff
@@ -176,3 +176,141 @@ func Test_ListPolicies(t *testing.T) {
 		assert.DeepEqual(t, expectedValue, actualValue)
 	})
 }
+
+func Test_GetPolicy(t *testing.T) {
```

Comment:
Can we change the test func names? The underscores are weird from a Go perspective. Func should just be named `TestGetPolicy`